src/types/state.js

```diff
@@ -283,6 +284,14 @@ export type UrlState = {|
   +profileSpecific: ProfileSpecificUrlState,
 |};
 
+/**
+ * Localization State
+ */
+export type L10nState = {|
+  +fetching: boolean,
```

Comment:
Also, we are usually using `is` + "the task" to keep the boolean values. It would be better to change this to `isFetching`.